_docs/_latest/use/cli/enforcer.md

```diff
@@ -17,40 +17,41 @@ Additional enforcement endpoints are in development.
 
 ## Before you begin
 
-Forseti Enforcer requires write permissions for the resources that it manages.
-When you setup Forseti using the installer, a service account is created and
+Enforcer requires write permissions for the resources that it manages.
+When you set up Forseti using the installer, a service account is created and
 granted write access to update firewalls on any project in your organization.
 
-If you did not use the installer, you did not enable enforcer during
-installation, or you want to run enforcer on a different instance than the
-Forseti server, then you need to grant the service account used by enforcer
-the following permissions:
+If you didn't use the installer, or you didn't enable Enforcer during
```

Comment:
@joecheuk is this a true statement? If someone uses the installer don't we add the Compute Security Admin role (is it called that?)?

Reply:
Yes, if someone uses the installer, the role `roles/compute.securityAdmin` will be granted to the forseti server service account during the installation process. Without the installer, user will have to manually assign that role to the service account in order for them to run the enforcer script.

Reply:
Do we need this paragraph then and shouldn't it be in the manual setup instead of here?

Reply:
Other than letting the user know the enforcer can only be run on the server instance (not CLI), the rest can be removed. Since we already have all the required roles listed in the manual installation guide so this might not be useful in the manual installation step neither.